.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest an idea for this project
+---
+
+<!--
+Thank you for suggesting an idea to make js-tools better.
+Please fill in as much of the template below as you're able.
+-->
+
+**Is this a new feature or an enhancement?**
+
+- [ ] **New feature**
+- [ ] **Enhancement**
```

Comment:
Interesting use of the checkboxes @p-netm, I like the concept though I'm not sure how I feel about these being automatically counted as `tasks` by GitHub, you know?

@moshthepitt - thoughts on this?

Reply:
@p-netm I agree with @cKellyDesign here.  Since these are not tasks (like below where you used checkboxes), Let's use numbers/bullets and ask the person creating the issue to delete the one that does not apply.